misc/services/acrn_manager/README.rst

```diff
@@ -157,6 +158,9 @@ You can see the available ``acrnd`` commands by running:
        ``RELEASE=1`` build)
```

Comment:
This value needs to be either 'y' or 'n' now, right?

Reply:
ACRN also has backward-compatibility for RELEASE=(0|1), this update is not related the acrnctl failure bug, if need, we can update this in another patch. Thanks!